### PR TITLE
AC Automated Test - LRAC-10739 | master

### PR DESCRIPTION
**Pull request by ViniLopes20**
https://issues.liferay.com/browse/LRAC-10740 (Ticket)
https://issues.liferay.com/browse/LRAC-10403 (Story)

### CODE REVIEW

**Comment by ViniLopes20** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/EmptyState.testcase [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+				ACUtils.viewGenericButton(buttonName = "Connect Data Source");
+			}
+		}
+
+		task ("Navigate to sites page") {
+			ACNavigation.goToSidebarOption(sidebarOption = "sites");
+		}
+
+		for (var tabName : list "Pages,Interests") {
+			task ("Switch to the another tab") {
```
Comment (manuferreira) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/EmptyState.testcase:69]: Hey Vini, I think here you can write `"switch to the other tab`" (same thing in the line 85, 133)
what do you think?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/EmptyState.testcase [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+				ACNavigation.switchTab(tabName = "${tabName}");
+			}
+
+			task ("Check page empty state message and the connect button") {
```
Comment (manuferreira) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/EmptyState.testcase:73]: Hey Vini, I think this task could be written like this `"Check the empty page status message and the connect button"` (same thing in line 89, 100, 111, 148)
what do you think?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/EmptyState.testcase [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+			ACNavigation.switchTab(tabName = "Interests");
+		}
+
+		task ("Check page empty state message and the connect button") {
+			ACUtils.viewGenericText(textValueList = "No Sites Synced from Data Sources,Connect a data source with sites data.");
+
+			ACUtils.viewGenericButton(buttonName = "Connect Data Source");
+		}
+
+		task ("Check the hyperlink taking user to the documentation") {
```
Comment (manuferreira) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/EmptyState.testcase:154]: Hey Vini, I think this one could be written like this `"Check if the hyperlink is taking the user to the documentation"`
what do you think?

**Review by manuferreira — CHANGES_REQUESTED** [review]

**Review by manuferreira — APPROVED** [review]

**Comment by ViniLopes20** [comment]
Hi @manuferreira, Thanks for reviewing this!

The code looks good to me!

**Comment by ViniLopes20** [comment]
ci:forward

**Pull request closed without merging**